frontend/src/modules/auth/actions.js

```diff
@@ -1,32 +1,29 @@
 import jwtDecode from "jwt-decode";
 
 import api from "../../utils/api";
-import {createNetworkActionTypes, networkAction} from "../../utils/actions";
-import {fetchInitialData} from "../shared/actions";
-import {constVal} from "../../utils/functions";
+import { createNetworkActionTypes, networkAction } from "../../utils/actions";
+import { fetchInitialData } from "../shared/actions";
```

Comment:
You have a dead import here. Your linter should be have picked that up. You can run the vscode command 'organize imports' to remove dead imports automatically on an open source file (cmd-shift-P opens the commands chooser in vscode).